code/modules/mob/mob.dm

```diff
@@ -52,6 +52,8 @@
 		move_intent = move_intents[1]
 	if(ispath(move_intent))
 		move_intent = decls_repository.get_decl(move_intent)
+	if(ispath(ai))
+		ai = new(src)
```

Comment:
This instance never appears to be deleted

Reply:
For some reason I thought only /atoms had to be qdel'd. Datums too?